src/ingest_validation_tools/directory-schemas/codex.yaml

```diff
@@ -1,65 +1,72 @@
-# A dataset consists of a single CODEX experimental run, and is
-# collected by the Akoya software into a single directory.
-#
-# directory beginning with drv_ contains analyzed files produced.
-# There are a few files in this directory that are used for the HuBMAP
-# data processing pipelines (channelNames.txt, experiment.json, segmentation.json)
-#
-# directory beginning with src_ contains the raw (unprocessed by Akoya) files.
+# A dataset consists of a single CODEX experimental run
 
 -
   pattern: '[^/]*NAV[^/]*\.tif'
   description: 'Navigational Image showing Region of Interest (Keyance Microscope only)'
   required: False
 -
   example: 'summary.pdf'
-  pattern: '.+\.pdf'
+  pattern: '[^/].*\.pdf'
   description: 'PDF export of Powerpoint slide deck containing the Image Analysis Report'
   is_qa_qc: True
+  required: False
 -
-  pattern: 'drv_[^/]+/channelNames\.txt'
-  description: 'Text file produced by the Akoya software which contains the (linearized) channel number and the Name/ID/Target of the channel (required for HuBMAP pipeline)'
+  pattern: '(raw|src_.*)/.*'
+  description: 'Raw data directory.'
   required: True
 -
-  pattern: 'src_[^/]+/experiment\.json'
-  description: 'JSON file produced by the Akoya software which contains the metadata for the experiment, including the software version used, microscope parameters, channel names, pixel dimensions, etc. (required for HuBMAP pipeline)'
+  pattern: '(processed|drv_.*)/.*'
+  description: 'Processed files produced by the Akoya software or alternative software.'
   required: True
 -
-  pattern: 'drv_[^/]+/experiment\.json'
-  description: 'JSON file produced by the Akoya software which contains the metadata for the experiment, including the software version used, microscope parameters, channel names, pixel dimensions, etc. (required for HuBMAP pipeline)'
+  pattern: '(raw|src_[^/]*)/exposure_times\.txt'
+  description: 'Comma separated text file used for background subtraction that contains valid exposure times for all cycles [e.g: Cycle,CH1,CH2,CH3,CH4].'
   required: False
 -
-  pattern: 'src_[^/]+/exposure_times\.txt'
-  description: 'Comma separated text file used for background subtraction that contains valid exposure times for all cycles [e.g: Cycle,CH1,CH2,CH3,CH4].'
+  pattern: 'raw/reg_00\.png'
+  description: 'Region overviews'
+  required: False
+-
+  pattern: '(raw|src_[^/]*)/[Ee]xperiment\.json'
+  description: 'JSON file produced by the Akoya software which contains the metadata for the experiment, including the software version used, microscope parameters, channel names, pixel dimensions, etc. (required for HuBMAP pipeline)'
```

Comment:
Explain where the variation in case comes from... In the future, will it be either one or the other?

Reply:
One data provider names it `experiment.json` and the other `Experiments.json` :shrug: